### ISSUE TITLE
geonode layer thumbnail error

### ISSUE THREAD

**Issue by hishamkaram** [GeoNode/geonode#4180]
save map error with remote layer:
```
Internal Server Error: /maps/new/data
Traceback (most recent call last):
  File "/Users/x/virtual_envs/geonode_env/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/x/Projects-Active/geonode/geonode/maps/views.py", line 707, in new_map_json
    map_obj.update_from_viewer(body, context={'request': request, 'mapId': map_obj.id, 'map': map_obj})
  File "/Users/x/Projects-Active/geonode/geonode/maps/models.py", line 222, in update_from_viewer
    self.save()
  File "/Users/x/virtual_envs/geonode_env/lib/python2.7/site-packages/polymorphic/models.py", line 82, in save
    return super(PolymorphicModel, self).save(*args, **kwargs)
  File "/Users/x/virtual_envs/geonode_env/lib/python2.7/site-packages/django/db/models/base.py", line 734, in save
    force_update=force_update, update_fields=update_fields)
  File "/Users/x/virtual_envs/geonode_env/lib/python2.7/site-packages/django/db/models/base.py", line 771, in save_base
    update_fields=update_fields, raw=raw, using=using)
  File "/Users/x/virtual_envs/geonode_env/lib/python2.7/site-packages/django/dispatch/dispatcher.py", line 189, in send
    response = receiver(signal=self, sender=sender, **named)
  File "/Users/x/Projects-Active/geonode/geonode/geoserver/signals.py", line 635, in geoserver_post_save_map
    create_gs_thumbnail(instance, overwrite=False)
  File "/Users/x/Projects-Active/geonode/geonode/geoserver/helpers.py", line 2003, in create_gs_thumbnail
    return implementation(instance, overwrite)
  File "/Users/x/Projects-Active/geonode/geonode/layers/utils.py", line 1027, in create_gs_thumbnail_geonode
    overwrite=overwrite, check_bbox=check_bbox)
  File "/Users/x/Projects-Active/geonode/geonode/layers/utils.py", line 974, in create_thumbnail
    image = _prepare_thumbnail_body_from_opts(request_body)
  File "/Users/x/Projects-Active/geonode/geonode/geoserver/helpers.py", line 1880, in _prepare_thumbnail_body_from_opts
    t_ll = mercantile.tile(bounds[0], bounds[1], zoom)
  File "/Users/x/virtual_envs/geonode_env/lib/python2.7/site-packages/mercantile/__init__.py", line 227, in tile
    "Y can not be computed for latitude {} radians".format(lat))
InvalidLatitudeError: Y can not be computed for latitude 2.80956967935 radians
```
request payload:
```
{
	"about": {
		"abstract": "us_cities_14a9cc0e",
		"title": "us_cities_14a9cc0e",
		"refresh_interval": 60000
	},
	"map": {
		"id": 0,
		"center": [-12613103.588416662, 5900239.27998824],
		"zoom": 3,
		"projection": "EPSG:3857",
		"layers": [{
			"opacity": 1,
			"wrapDateLine": true,
			"displayOutsideMaxExtent": true,
			"name": "mapnik",
			"visibility": true,
			"source": 0,
			"group": "background",
			"fixed": true,
			"type": "OpenLayers.Layer.OSM",
			"id": 1
		}, {
			"opacity": 1,
			"crs": {
				"type": "name",
				"properties": "EPSG:4269"
			},
			"wrapDateLine": true,
			"selected": true,
			"attribution": "<span class='gx-attribution-title'>Administrator Exchange</span>",
			"name": "geonode:us_cities_14a9cc0e",
			"format": "image/png",
			"title": "us_cities_14a9cc0e",
			"visibility": true,
			"capability": {
				"styles": [{
					"title": "us_cities_14a9cc0e",
					"legend": {
						"height": "40",
						"width": "22",
						"href": "http://localhost/geoserver/ows?service=wms&request=GetLegendGraphic&format=image%2Fpng&width=20&height=20&layer=geonode%3Aus_cities_14a9cc0e",
						"format": "image/png"
					},
					"name": "us_cities_14a9cc0e"
				}],
				"attribution": {
					"title": "Administrator Exchange"
				},
				"name": "geonode:us_cities_14a9cc0e",
				"infoFormats": ["text/plain", "application/vnd.ogc.gml", "text/xml", "application/vnd.ogc.gml/3.1.1", "text/xml; subtype=gml/3.1.1", "text/html", "application/json"],
				"abstract": "No abstract provided",
				"title": "us_cities_14a9cc0e",
				"srs": {
					"EPSG:3857": true
				},
				"prefix": "geonode",
				"bbox": {
					"EPSG:900913": {
						"srs": "EPSG:900913",
						"bbox": [-17570076.3321, 2238543.24293, -7656130.84475, 9561935.31678]
					},
					"EPSG:4326": {
						"srs": "EPSG:4326",
						"bbox": [-157.834681123, 19.7086074721, -68.7761935506, 64.8231249481]
					},
					"EPSG:4269": {
						"srs": "EPSG:4269",
						"bbox": [-157.834681122551, -68.7761935505984, 19.7086074727151, 64.8231249488412]
					},
					"EPSG:3857": {
						"srs": "EPSG:3857",
						"bbox": [-17570076.3321, 2238543.24293, -7656130.84475, 9561935.31678]
					}
				},
				"formats": ["image/png", "application/atom xml", "application/atom+xml", "application/json;type=utfgrid", "application/openlayers", "application/pdf", "application/rss xml", "application/rss+xml", "application/vnd.google-earth.kml", "application/vnd.google-earth.kml xml", "application/vnd.google-earth.kml+xml", "application/vnd.google-earth.kml+xml;mode=networklink", "application/vnd.google-earth.kmz", "application/vnd.google-earth.kmz xml", "application/vnd.google-earth.kmz+xml", "application/vnd.google-earth.kmz;mode=networklink", "atom", "image/geotiff", "image/geotiff8", "image/gif", "image/gif;subtype=animated", "image/jpeg", "image/png8", "image/png; mode=8bit", "image/svg", "image/svg xml", "image/svg+xml", "image/tiff", "image/tiff8", "image/vnd.jpeg-png", "kml", "kmz", "openlayers", "rss", "text/html; subtype=openlayers", "utfgrid"],
				"keywords": ["features", "us_cities_14a9cc0e"],
				"storeType": "dataStore",
				"queryable": true,
				"llbbox": [-157.834681123, 19.7086074721, -68.7761935506, 64.8231249481]
			},
			"srs": "EPSG:3857",
			"bbox": [-17570076.3321, 2238543.24293, -7656130.84475, 9561935.31678],
			"getFeatureInfo": {
				"fields": ["uident", "popclass", "name", "capital", "stateabb", "country"],
				"propertyNames": {
					"stateabb": null,
					"name": null,
					"country": null,
					"popclass": null,
					"uident": null,
					"capital": null
				}
			},
			"fixed": false,
			"source": 2,
			"schema": [{
				"name": "uident",
				"visible": true
			}, {
				"name": "popclass",
				"visible": true
			}, {
				"name": "capital",
				"visible": true
			}, {
				"name": "stateabb",
				"visible": true
			}, {
				"name": "country",
				"visible": true
			}, {
				"name": "wkb_geometry",
				"visible": false
			}]
		}]
	},
	"sources": [{
		"ptype": "gxp_osmsource",
		"use_proxy": true
	}, {
		"url": "//localhost/geoserver/wms",
		"restUrl": "/gs/rest",
		"ptype": "gxp_wmscsource",
		"title": "Local Geoserver",
		"isVirtualService": false
	}, {
		"url": "//localhost/geoserver/ows",
		"restUrl": "/gs/rest",
		"ptype": "gxp_wmscsource",
		"isVirtualService": false
	}, {
		"url": "http://try.cartoview.net/geoserver/wms",
		"title": "[R] GeoNode Try Server",
		"remote": true,
		"ptype": "gxp_wmscsource",
		"name": "trygeonode-local-geoserver",
		"isVirtualService": false
	}, {
		"ptype": "gxp_bingsource",
		"name": "Bing",
		"defaultServer": true
	}, {
		"ptype": "gxp_mapquestsource",
		"name": "MapQuest",
		"defaultServer": true
	}, {
		"ptype": "gxp_mapboxsource",
		"name": "MapBox",
		"defaultServer": true
	}, {
		"ptype": "gxp_arcrestsource",
		"name": "Esri",
		"proj": "EPSG:4326",
		"defaultServer": true,
		"alwaysAnonymous": true,
		"url": "https://services.arcgisonline.com/arcgis/rest/services/NGS_Topo_US_2D/MapServer/"
	}]
}
```
geonode version: 2.8.x branch

**Comment by hishamkaram** [comment]
here is the us_cities Layer [download](http://try.cartoview.net/geoserver/wfs?format_options=charset%3AUTF-8&typename=geonode%3Aus_cities&outputFormat=SHAPE-ZIP&version=1.0.0&service=WFS&request=GetFeature&access_token=7d180b4a209611e987b80242ac130004)

**Comment by afabiani** [comment]
Fixed on master / 2.10 branches
